plugins/modules/zos_unarchive.py

```diff
@@ -36,7 +36,7 @@
       - I(src) can be a USS file or MVS data set name.
       - USS file paths should be absolute paths.
       - MVS data sets supported types are C(SEQ), C(PDS), C(PDSE).
-      - GDS relative names are supported C(e.g. USER.GDG(-1)).
+      - GDS relative names are supported C(e.g. USER.GDG(-1\)).
```

Comment:
This is a raw string , is it necessary to escape the closing paren:
```
(-1\).)
```
Could this be the source of some of the hanging closing parens we saw during doc generation.

Reply:
fixed